tests/end2end/playwright/globals.js

```diff
@@ -99,11 +110,10 @@ export async function reloadMap(page, check = true) {
 
 
 /**
- * editedFeatureIds function
  * Get the last IDs when saving features as JSON
  * @param {Page} page The page object
  *
- * @return {JSON} The JSON response, like {"id":"31"}
+ * @return {Promise<any>} The JSON response, like {"id":"31"}
```

Comment:
Same question

Reply:
Same response, `JSON.parse` returns `any` and not `JSON`. It could be an object, an array, a number or a string.